cmd/lakectl/cmd/common_helpers.go

```diff
@@ -267,3 +270,47 @@ func MustParsePathURI(name, s string) *uri.URI {
 	}
 	return u
 }
+
+func IsValidAccessKeyID(accessKeyID string) bool {
+	return (len(accessKeyID) == 20 &&
+		(strings.HasPrefix(accessKeyID, "ASIA") || strings.HasPrefix(accessKeyID, "AKIA")) &&
+		(string(accessKeyID[4]) == "I" || string(accessKeyID[4]) == "J") &&
+		(strings.HasSuffix(accessKeyID, "A") || strings.HasSuffix(accessKeyID, "Q")) &&
+		IsContainOnlyUpperLettersOrNumber(accessKeyID))
+}
```

Comment:
@idanovo - Thank you for fixing this.
As I was writing my comment on ```go IsContainOnlyUpperLettersOrNumber```, I started wondering about regexp and alike, and I think this entire function can be reduced to a single regexp match.

``` go
var accessKeyRegexp = regexp.MustCompile(`^A[K|S]IA[I|J][A-Z0-9]{14}[A|Q]$`)

func IsValidAccessKeyID(accessKeyID string) bool {
	return accessKeyRegexp.MatchString(accessKeyID)
}
```
I'm not sure if it makes the code more readable ot less. Guess it is a matter of personal preferation. Up to you :)

Reply:
you are right! changed